drivers/SmartThings/matter-switch/src/test/test_matter_switch_device_types.lua

```diff
@@ -195,7 +195,88 @@ local mock_device_parent_child_switch_types = test.mock_device.build_test_matter
   }
 })
 
+local mock_device_parent_child_different_types = test.mock_device.build_test_matter_device({
+  label = "Matter Switch",
+  profile = t_utils.get_profile_definition("matter-thing.yml"),
```

Comment:
nit: make this switch binary just to match what this device would join as more realistically

Reply:
Updated as suggested in latest commit